### PR TITLE
Pin minikube to 1.25.2 in shell.nix

### PR DESCRIPTION
**Pull request by mlieberman85**
Tekton doesn't support kubernetes >=1.25 and minikube >1.25.2 switches to kubernetes version >=1.25.

Signed-off-by:  Michael Lieberman <mlieberman85@gmail.com>

### CODE REVIEW

File: shell.nix [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+    rev = "v${version}";
+    sha256 = "sha256-WIk4ibq7jcqao0Qiz3mz9yfHdxTUlvtPuEh4gApSDBg=";
+  };
+  in (pkgs.minikube.override rec {
+    buildGoModule = args: pkgs.buildGoModule.override {} (args // {
+      inherit src version;
+      vendorSha256 = "sha256-8QqRIWry15/xwBxEOexMEq57ol8riy+kW8WrQqr53Q8=";
+    });
+  });
+in with pkgs;
```
Comment (bradbeck) [inline shell.nix:17]: When this built locally it emitted the following:
```
go build  -tags "" -ldflags="-X k8s.io/minikube/pkg/version.version=v1.25.2 -X k8s.io/minikube/pkg/version.isoVersion=v1.25.2 -X k8s.io/minikube/pkg/version.gitCommitID=v1.27.0 -X k8s.io/minikube/pkg/version.storageProvisionerVersion=v5" -o out/minikube k8s.io/minikube/cmd/minikube
```
Which seems a little odd with `gitCommitID=v1.27.0`

Comment (mlieberman85) [reply shell.nix:17]: oh weird. I'll take a look.

**Review by bradbeck — APPROVED** [review]

**Review by bradbeck — CHANGES_REQUESTED** [review]
Probably want to remove the empty file `foo`

**Comment by mlieberman85** [comment]
> Probably want to remove the empty file `foo`

oops

**Review by bradbeck — APPROVED** [review]

**Review by pxp928 — APPROVED** [review]
LGTM

**Pull request merged**